scripts/appleseed.package.py

```diff
@@ -750,9 +935,34 @@ def get_dependencies_for_file(self, filepath, fix_paths=True, verbose=True):
             lib = m.group(1)
 
             # Ignore self-references (why do these happen?).
-            if lib == filename:
+            if lib == filepath:
```

Comment:
This is a significant change, did you check that it still works?

Reply:
I'll double check if this is correct, I know why these self references happen so I'll update the comments too.

Reply:
Did you mean to fix/update the code and comment here? As far as I can tell, the situation hasn't changed since I've made the comment that this was a significant change.